### PR TITLE
JSON file was updated,Changes to the edit and delete were made,most o…

### PR DESCRIPTION
**Pull request by gokilakittusamy**
…f the testcases were automated,consturctor was used,data were bind in a an object to pass between the methods,number of line in the code were reduced,separate pages for education and certificate were created.

### CODE REVIEW

**Comment by gokilakittusamy** [comment]
Changes were made based on the first feedback from the mentor.

File: Pages/ProfileCertificatePage.cs [hunk 1]
```diff
@@ -96,10 +94,13 @@ public JsonLibHelper.ReturnObjContainer EnterEditCertificate(JsonLibHelper.Certi
             return new ReturnObjContainer(ReturnStatus, ReturnMessage);
         }
 
-        public object DeleteEducation(JsonLibHelper.CertificateData item)
+        public JsonLibHelper.ReturnObjContainer DeleteCertificate(JsonLibHelper.CertificateData item)
         {
             driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody/tr/td[4]/span[2]/i")).Click();
```
Comment (amanbansalnz) [inline Pages/ProfileCertificatePage.cs:99]: your driver.FindElement should follow the POM pattern can you please change this everywhere you should now be familiar  with this due to your onboarding task

Comment (amanbansalnz) [reply Pages/ProfileCertificatePage.cs:99]: also the way you are asserting is not correct you should assert that the value is present in the table and also not present, in regards to the message you get the text from the notification message and assert that is matches your expected everything else is fine

Comment (gokilakittusamy) [reply Pages/ProfileCertificatePage.cs:99]: Hi Aman,
I hope this message finds you well. I wanted to extend my sincerest apologies for the delay in submitting the task. Unfortunately, I encountered some health issues recently, including a miscarriage, which led to a period of depression. Despite these challenges, I managed to submit the task now and am committed to catching up on the following.
Based on your review, I have made the following changes:
1.Followed POM pattern in finding the element.
2.Changed the way I used assertion(have included the success/ error messages).
Hope I have understood and implemented all the remarks you mentioned earlier.
Thank you for your understanding and support during this time.
Gokila

**Pull request merged**